package.json

```diff
@@ -3,7 +3,13 @@
   "version": "0.1.0",
   "repository": "https://github.com/openedx/edx-platform",
   "scripts": {
-    "postinstall": "scripts/copy-node-modules.sh"
+    "postinstall": "scripts/copy-node-modules.sh",
+    "build": "echo 'WARNING: `npm run build` in edx-platform is experimenal. Use at your own risk.' && npm run webpack && npm run compile-sass",
+    "build-dev": "echo 'WARNING: `npm run build-dev` in edx-platform is experimenal. Use at your own risk.' && npm run webpack-dev && npm run compile-sass-dev",
+    "webpack": "NODE_ENV=${NODE_ENV:-production} \"$(npm bin)/webpack\" --progress --config=${WEBPACK_CONFIG_PATH:-webpack.prod.config.js}",
+    "webpack-dev": "NODE_ENV=development \"$(npm bin)/webpack\" --progress --config=webpack.dev.config.js",
+    "compile-sass": "scripts/compile_sass.py --env=${NODE_ENV:-production}",
+    "compile-sass-dev": "scripts/compile_sass.py --env=development"
```

Comment:
I'd love to add comments here, but I can't since it's JSON. I do plan to document these in edx-platform's README, though, once they stabilize more.

In short:
* `npm run webpack` is a super thin wrapper around `webpack`.
* `npm run compile-sass` calls a script (currently written in Python, maybe one day written in Bash) which compiles Sass into CSS.
* `npm run build` is a combination of the above.
* The `-dev` targets are just simple variations of the base commands, for the convenience of developres.
* Wherever it was possible, I've written these so that:
  * You can override them with en vars, eg `WEBPACK_CONFIG_PATH=my-special-config.js npm run webpack`
  * You can provide extra options: `npm run compile-sass -- --theme-dir=/my/cool/themes`